pkg/steps/bundle_source_test.go

```diff
@@ -49,11 +50,21 @@ func TestReplaceCommand(t *testing.T) {
 	if err := exec.Command("cp", "-a", "testdata/4.6", temp).Run(); err != nil {
 		t.Fatalf("Failed to copy testdata to tempdir: %v", err)
 	}
+	originalDir, err := os.Getwd()
```

Comment:
Prefer to pass the working dir to `exec.Command()` instead of changing global state